cmd/tuf/app/add-delegation.go

```diff
@@ -24,6 +24,7 @@ import (
 	"path/filepath"
 	"strings"
 
+	csignature "github.com/sigstore/cosign/pkg/signature"
```

Comment:
Aside (and way out of scope of this PR, we've been using `cosign/pkg/signature` for some time) – is this a candidate for sigstore/sigstore? Intuitively, anything root-signing needs should be available from a lower level library than cosign. 🤔

Reply:
https://github.com/sigstore/cosign/blob/main/pkg/signature/annotations.go would just be for Cosign. The package will need some refactoring

Reply:
Yeah :/ I think the reason this was in here was because cosign was handling some "meta" signer/verifier methods that parsed keys from KMS references OR files, which is the method we want. sigstore/sigstore only made the individual funcs available.

Reply:
I'll make a general tracking issue to remove the dependency on sigstore/cosign

@znewman01 FYI: another point in favor of a better sigstore go library away from cosign.